scripts/ci/run_perftest.sh

```diff
@@ -90,24 +125,31 @@ systemctl unmask influxdb.service
 systemctl start influxdb
 
 # Common variables used across all tests
-datestring=$(date +%s)
+datestring=${TEST_COMMIT_TIME}
 seed=$datestring
 db_name="benchmark_db"
 
+# Controls the cardinality of generated points. Cardinality will be scale_var * scale_var.
+scale_var=1000
+
+# How many queries to generate.
+queries=500
+
+# Lines to write per request during ingest
+batch=5000
+# Concurrent workers to use during ingest/query
+workers=4
+
+# How long to run each set of query tests. Specify a duration to limit the maximum amount of time the queries can run,
+# since individual queries can take a long time.
+duration=30s
+
 # Helper functions containing common logic
 bucket_id() {
   bucket_id=$(curl -H "Authorization: Token $TEST_TOKEN" "http://${NGINX_HOST}:8086/api/v2/buckets?org=$TEST_ORG" | jq -r ".buckets[] | select(.name | contains(\"$db_name\")).id")
   echo $bucket_id
 }
 
-clean_db () {
```

Comment:
So doesn't omitting the DB cleanup between tests mean that the sequence of the tests could skew results? i.e. the script later loops over `iot` and `metaquery` to do ingest tests without deleting the DB in between. It seems like those are not totally comparable tests, as one is being measured "cold" and the other measured "hot." Thoughts?

Reply:
I think we're ok because we're still shutting down the DB after each ingest run to force compaction, so each test runs starts off cold. AFAIK it should be fine for the iot and metaquery data to both be in the bucket while we run query tests, since they're in different measurements. There might be some slowdown across the board from scanning extra indices but I'd expect it to impact both tests equally.

The query tests don't have any sort of restart logic in-between suites at the moment. I can add that so we know each set of queries starts against a cold DB.